nUpdate/Core/Localization/de-AT.json

```diff
@@ -1,52 +0,0 @@
-{  
```

Comment:
Yes, but then we would need to do this for every culture. Also with that you can make small changes for a country if you want. But you could do that so that it does check if the specific culture exists (e. g. de-AT) and if not, it should check if de exists and then use that.